build/ps-entrypoint.sh

```diff
@@ -126,7 +126,7 @@ _get_tmpdir() {
 
 	tmpdir_path=$(_get_cnf_config mysqld tmpdir "")
 	if [[ -z ${tmpdir_path} ]]; then
-		tmpdir_path=$(_get_cnf_config xtrabackup tmpdir "")
+		tmpdir_path=$(_get_cnf_config mysqld tmpdir "")
```

Comment:
why this change needed?

Reply:
@hors ^^

Reply:
it was misprint, I have fixed it 
@delgod  thank's